packages/runtime/src/components/core/HTML.tsx

```diff
@@ -0,0 +1,40 @@
+import { Type } from '@sinclair/typebox';
+import { cx } from '@emotion/css';
+import React from 'react';
+import { implementRuntimeComponent } from '../../utils/buildKit';
+
+export default implementRuntimeComponent({
+  version: 'core/v1',
+  metadata: {
+    name: 'html',
+    displayName: 'HTML',
+    description: '',
+    isDraggable: false,
+    isResizable: false,
+    exampleProperties: {
+      html: '',
+    },
+    exampleSize: [1, 1],
+    annotations: {
+      category: 'Display',
+    },
+  },
+  spec: {
+    properties: Type.Object({
+      html: Type.String(),
+    }),
+    state: Type.Object({}),
+    methods: {},
+    slots: {},
+    styleSlots: ['content'],
+    events: [],
+  },
+})(({ elementRef, customStyle, html }) => {
+  return (
+    <div
+      ref={elementRef}
+      className={cx(customStyle?.content)}
+      dangerouslySetInnerHTML={{ __html: html }}
```

Comment:
worth a try?
https://github.com/aknuds1/html-to-react

Reply:
Perhaps we'd better move this component to the project, not in runtime.

Reply:
The html components have been removed and will be placed in a custom component library